Source/WebCore/style/StyleResolver.cpp

```diff
@@ -544,10 +544,10 @@ std::unique_ptr<RenderStyle> Resolver::defaultStyleForElement(const Element* ele
 
 Vector<RefPtr<const StyleRule>> Resolver::styleRulesForElement(const Element* element, unsigned rulesToInclude)
 {
-    return pseudoStyleRulesForElement(element, PseudoId::None, rulesToInclude);
+    return pseudoStyleRulesForElement(element, std::nullopt, rulesToInclude);
```

Comment:
Using default initialization `{ }` is more future-proof than explicit `std::nullopt`

Reply:
(same comment for all of them obviously)